platform.md

```diff
@@ -948,8 +948,10 @@ The launcher:
 ### Run Image Resolution
 
 Given stack metadata containing `run-image.image` and a set of `run-image.mirrors`. The `<run-image>` for a given `<image>` shall be resolved as follows:
-- **If** any of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, this value will become the `<run-image>`
-- **If** none of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>`, `<run-image.image>` will become the `<run-image>`
+
+**If** any of `run-image.image` or `run-image.mirrors` is accessible with read permissions and has a registry matching that of `<image>`, this value will become the `<run-image>`.
```

Comment:
nit: I think this sentence would be easier to parse it was phrase like:

"**If** any of `run-image.image` or `run-image.mirrors` has a registry matching that of `<image>` and is accessible with read permissions, this value will become `<run-image>`.

The phrasing now, kind of buries the lead. I had to read it a few times to understand how it was different from the point below.